src/app/[lang]/(protected)/chat/[chatId]/chatSearch.tsx

```diff
@@ -0,0 +1,45 @@
+"use client";
+
+import {ChangeEvent } from "react";
+import Image from "next/image";
+import searchIcon from "public/search_icon.svg";
+import cancelIcon from "public/cancel_icon.svg";
+
+export default function ChatSearch({
+    searchValue,
+    searchValueHandler,
+    isClearBtnActive,
+    clearSearch,
+    searchDisableHandler
+} : {
+    searchValue: string,
+    searchValueHandler: (event: ChangeEvent<HTMLInputElement>) => void,
+    isClearBtnActive: boolean,
+    clearSearch: () => void,
+    searchDisableHandler: () => void
+}) {
```

Comment:
To suppress this warning remove the `use client`

`Props must be serializable for components in the "use client" entry file, "searchValueHandler" is invalid.`